site/en/docs/devtools/search/index.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Search: Find text across all loaded resources"
+authors:
+  - sofiayem
+date: 2022-05-27
+#updated: YYYY-MM-DD
+description: "Find text across all loaded resources with the Search tab."
+---
+
+Use the **Search** tab to find text across all loaded resources.
+
+{% Aside 'gotchas' %}
+The **Search** tab doesn't show results from network headers and responses. To search across them, follow the steps in [Search network headers and responses](/docs/devtools/network/#search).
+{% endAside %}
+
+Alternatively, use [built-in search bars](#built-in-search-bars) to find text in a resource opened in a specific tool.
+
+## Open the Search tab {: #open-search }
+
+You can open the **Search** tab in several ways. First, [open DevTools](/docs/devtools/open/), then do one of the following:
+
+- In the top-right corner of DevTools, select {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Customize and control DevTools", width="20", height="20" %} **Customize and control DevTools** > **More tools** > **Search**.
+- Press `Esc` to open [**Drawer**](/docs/devtools/customize/#drawer), and in the top-left corner, select {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Customize and control DevTools", width="20", height="20" %} **More Tools** > **Search**.
```

Comment:
I wonder if we can use the `<kbd>` element here for better semantics? Or maybe we don't style that appropriately on d.c.c. Just a thought.